receiver/sqlqueryreceiver/metrics.go

```diff
@@ -46,6 +48,23 @@ func rowToMetric(row metricRow, cfg MetricCfg, dest pmetric.Metric) error {
 	return nil
 }
 
+func setTimestamp(cfg MetricCfg, dest pmetric.Metric, ts pcommon.Timestamp, scrapeCfg scraperhelper.ScraperControllerSettings) pmetric.NumberDataPoint {
+	dataPointSlice := setMetricFields(cfg, dest)
+	dp := dataPointSlice.AppendEmpty()
```

Comment:
These two actions are shouldn't be part of this function. They have nothing to do with timestamps

Reply:
Moved them back to previous method.